app/main.py

```diff
@@ -2,5 +2,19 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args, **kwargs) -> None:
+        key = args
```

Comment:
indeed, on line 8 you can do without "key"

Reply:
Yeah, it was a silly thing ))) Goin' to correct ;)